podcasts/Sharing/Clip/VideoExporter.swift

```diff
@@ -192,13 +192,20 @@ enum VideoExporter {
         exportSession.outputFileType = fileType
         exportSession.timeRange = CMTimeRange(start: .zero, duration: CMTime(seconds: duration, preferredTimescale: 600))
 
+        let timer = Timer(timeInterval: 0.01, repeats: true) { _ in
+            progress.completedUnitCount = Int64(exportSession.progress * 100)
+        }
+        RunLoop.main.add(timer, forMode: .common)
```

Comment:
This needs to be added manually. This method typically runs in a background queue so `scheduledTimer` will be on the wrong thread.